### ISSUE TITLE
RFE: Automatically set nodatacow on $pooldir when on btrfs

### ISSUE THREAD

**Issue by cmurf** [os-autoinst/openQA#4679]
Summary:  Btrfs upstream advises VM images use nodatacow, therefore make sure `chattr +C` is set on the openqa directory (at create time).

Background: On (open)SUSE, `/var` has nodatacow file attribute `C` set at installation time. But most other distros don't do this, including Fedora. Meanwhile, libvirt now checks if a dir pool is on btrfs and by default sets the nodatacow file attribute.
https://listman.redhat.com/archives/libvir-list/2020-July/205982.html

Since libvirt isn't being used here, I think it's reasonable to just unconditionally set nodatacow file attribute on btrfs whenever creating a new parent directory to be used for openqa. Any new dirs and files, including qcow2, will inherit this file attribute.

**Comment by okurz** [comment]
Reopening as likely still valid. Also I created https://progress.opensuse.org/issues/117136 to prevent the stale bot to close issues which are still valid.

**Comment by kalikiana** [comment]
> Reopening as likely still valid. Also I created https://progress.opensuse.org/issues/117136 to prevent the stale bot to close issues which are still valid.

It doesn't look like this worked. At least this issue was still marked as stale. 🤔

**Comment by okurz** [comment]
yes, but the PR is only marked as stale, not closed. That's what we wanted.
